.github/workflows/continuous-integration.yml

```diff
@@ -1232,3 +1232,47 @@ jobs:
           channel_id: ${{ env.VETS_WEBSITE_CHANNEL_ID }}
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+  # Please keep steps-evaluation at the end of the workflow as it's meant to evaluate the entirety of the workflow steps that have run.
+  steps_evaluation:
+    runs-on: ubuntu-latest
+    name: Steps Evaluation
+    needs:
```

Comment:
Were `unit-tests` and `contract-tests` intentionally left out?

Reply:
I only left them out as I'm trying to ensure that it runs at the end, and cypress tests traditionally have taken longer to finish. That said, I can add them for security just in case there's an edge case where test selection makes cypress too quick.

Reply:
Sounds like adding them in won't be a problem, and should cover all bases just in case.